packages/mui-material/src/ListItemButton/ListItemButton.js

```diff
@@ -58,6 +58,7 @@ const ListItemButtonRoot = styled(ButtonBase, {
   alignItems: 'center',
   position: 'relative',
   textDecoration: 'none',
+  width: "100%",
```

Comment:
```suggestion
  minWidth: 0,
```
I rather use `minWidth` in this case. We should not use `width` with flex-box unless for a specific reason.